src/Illuminate/Database/Connectors/Connector.php

```diff
@@ -84,6 +85,23 @@ public function setDefaultOptions(array $options)
     }
 
     /**
+     * Returns the class name for the PDO connection to use.
+     *
+     * If doctrine/dbal is installed it will be `\Doctrine\DBAL\Driver\PDOConnection`,
+     * else `PDO`.
+     *
+     * @return string
+     */
+    public function getPdoClass()
+    {
+        if (class_exists('Doctrine\DBAL\Driver\PDOConnection')) {
+            return \Doctrine\DBAL\Driver\PDOConnection::class;
```

Comment:
But DBAL may not be installed. Doesn't it throw an error if the class does not exist?